mei/integration.py

```diff
@@ -36,7 +37,7 @@ def load(self, key):
             return self._load_model(key)
         if not self._is_cached(key):
             self._cache_model(key)
-        return self._get_cached_model(key)
```

Comment:
Could you make a suggestion how the test could look like? I am unsure here how to do this. In general, as the model is saved as reference in the Cache, downstream code could (accidentally) change the model's attributes. This is prevented by `deepcopy`.

Reply:
I agree. This could be tested by loading a model, changing an attribute on it, loading it again an asserting that the second one still has the original value.